### PR TITLE
chore: adds sample.env vars and updates readme to ease dev onboarding

### PR DESCRIPTION
**Pull request by 0xean**
## Description

Makes things that are not secret, not so hard to find for new developers who are contributing. 

Specifically the `REACT_APP_PORTIS_DAPP_ID` and `REACT_APP_ETHEREUM_NODE_URL`

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. etc...

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -8,5 +8,5 @@ REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=https://dev-api.ethereum.shapeshift.com
 REACT_APP_UNCHAINED_ETHEREUM_WS_URL=wss://dev-api.ethereum.shapeshift.com
 REACT_APP_UNCHAINED_BITCOIN_HTTP_URL=https://dev-api.bitcoin.shapeshift.com
 REACT_APP_UNCHAINED_BITCOIN_WS_URL=wss://dev-api.bitcoin.shapeshift.com
-REACT_APP_PORTIS_DAPP_ID=<get dapp ID from someone>
-REACT_APP_ETHEREUM_NODE_URL=<get free key from infura or other provider>
+REACT_APP_PORTIS_DAPP_ID=8609e6a8-e0dc-45e6-a0ad-edde63a4cdda
+REACT_APP_ETHEREUM_NODE_URL=https://eth-mainnet.alchemyapi.io/v2/lG_Rc1WYrmqrOUzTigl1CcSqlVAesAJl
```
Comment (0xdef1cafe) [inline sample.env:12]: portis matches prod, we use an infura url starting with `d7` in prod - do we wanna use that? anyone can `ctrl-f` it in the prod build files if they want.

**Review by 0xdef1cafe — COMMENTED** [review]
question about eth node url

File: sample.env (hunk 1 above)
Comment (0xean) [reply sample.env]: done

**Review by 0xdef1cafe — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -63,26 +63,6 @@ On Linux and MacOS it works out of the box following the steps.<br/>
...
-  Needed for certain Defi integrations such as Yearn; the app will malfunction when connecting a wallet without it.
-
-  Any Ethereum node should do, but you can get your own node URL for testing by doing the following:
-
-  1. Go to https://infura.io/dashboard
-  2. Set up a free account
-  3. Make a new project
-
-      Your key should use "JSON-RPC over HTTPS" and look like this: `https://mainnet.infura.io/v3/<your project id>`
-
```
Comment (mrnerdhair) [inline README.md:85]: maybe we just drop the line `Some of these variables are deployment-specific, so they aren't included in the repository` and add a reference to env.sample? This is still good info, and if someone's deploying something somewhere that isn't app.shapeshift.com or localhost, they'll need their own keys

**Review by mrnerdhair — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (0xean) [reply README.md:85]: will make this change

**Pull request merged**